cmd/occollector/app/collector/collector.go

```diff
@@ -93,14 +96,17 @@ func (app *Application) execute() {
 		os.Exit(1)
```

Comment:
nit: maybe not for this PR, but we should be consistent in our use of `log.Fatalf` vs log followed by os.Exit